src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import MigrationPolicyEmptyListImage from 'images/migrationPoliciesEmptyState.png';
```

Comment:
```suggestion
import {ReactComponent as MigrationPolicyEmptyListImage} from 'images/migrationPoliciesEmptyState.png';
```

can try to see if its working for u? its working for svgs, not sure about pngs

Reply:
tried to do it and it doesn't work for me